src/components/Navigation/Navigation.jsx

```diff
@@ -1,30 +1,28 @@
 import React from "react";
 import './Navigation.scss';
-
+import { routes } from "../../routes";
 
 class Navigation extends React.Component{
-
     showHiddenMenu = (e) => {
-        // document.querySelector("#hidden-menu").className === "nav-column-hidden" ? 
-        // console.log("on ma klase nav-column-hidden") : 
+        // document.querySelector("#hidden-menu").className === "nav-column-hidden" ?
+        // console.log("on ma klase nav-column-hidden") :
         // console.log("NIE ma klasy nav-colum-hidden");
 
+        // Pobądź się tych query selectorów - classnames na pomoc! :)
```

Comment:
Selektory do wyrzucenia ;) To chyba póki co najbardziej "kłuje" mnie w oczy, bo reszta to kwestia Twojego doświadczenia, a to jest reactowy antipattern :)